server/services/userService.ts

```diff
@@ -38,15 +40,15 @@ export default class UserService {
 
   async getUsers(
     token: string,
+    region?: string,
```

Comment:
I think it would make sense to follow the pattern of the other params in this function and give region a default value